Assets/UI/worldinput_CQUI.lua

```diff
@@ -0,0 +1,559 @@
+-- ===========================================================================
+-- Base File
+-- ===========================================================================
+include("WorldInput");
+
+-- ===========================================================================
+-- Cached Base Functions
+-- ===========================================================================
+BASE_CQUI_UpdateDragMap            = UpdateDragMap;
+BASE_CQUI_RealizeMovementPath      = RealizeMovementPath;
+BASE_CQUI_OnUnitSelectionChanged   = OnUnitSelectionChanged;
+BASE_CQUI_OnInputActionTriggered   = OnInputActionTriggered;
+BASE_CQUI_DefaultKeyDownHandler    = DefaultKeyDownHandler;
+BASE_CQUI_DefaultKeyUpHandler      = DefaultKeyUpHandler;
+BASE_CQUI_OnDefaultKeyDown         = OnDefaultKeyDown;
+BASE_CQUI_OnDefaultKeyUp           = OnDefaultKeyUp;
+BASE_CQUI_OnPlacementKeyUp         = OnPlacementKeyUp;
+BASE_CQUI_ClearAllCachedInputState = ClearAllCachedInputState;
+BASE_CQUI_LateInitialize           = LateInitialize;
+BASE_CQUI_Initialize               = Initialize;
+
+-- ===========================================================================
+-- CQUI Members
+-- ===========================================================================
+local CQUI_HOTKEYMODE_ENHANCED    = 2;
+local CQUI_HOTKEYMODE_CLASSIC     = 1;
+local CQUI_HOTKEYMODE_STANDARD    = 0;
+
+local m_isInputBlocked  :boolean = false;
```

Comment:
Does CQUI need this variable? Unmodded version removed some usages two patches ago in September patch and completely removed it in the last patch.

Reply:
not sure - didn't investigate, but if not, then I'm ok to remove it.  I'll look and see...

Reply:
I think you're right, will remove it.